src/main/scala/de/heikoseeberger/reactiveflows/MessageEventPublisher.scala

```diff
@@ -24,6 +24,7 @@ object MessageEventPublisher {
   def props(mediator: ActorRef, bufferSize: Int) = Props(new MessageEventPublisher(mediator, bufferSize))
 }
 
+// TODO: collapse with FlowEventPublisher
```

Comment:
Not possible, since `Flow.MessageEvent` and `FlowFacade.FlowEvent` don't (and shall not) share a common super type. Yet it would be possible to collapse (merge) the `Source`s created from these publishers. Of course the event types must then contain information about the type, e.g. "flow-added" instead of "added" and the JS must be adjusted. That might be a handy simplification.